pom.xml

```diff
@@ -6,9 +6,9 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.marklogic</groupId>
   <artifactId>marklogic-mule-connector</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>2.0.1-SNAPSHOT</version>
```

Comment:
I'll deal with "SNAPSHOT" later. Right now, I'm still using different values to test in Anypoint Studio.